### ISSUE TITLE
consider readme or cli

### ISSUE THREAD

**Issue by satra** [sensein/etelemetry-client#12]
sometimes it may be useful to embed the check functionality as a cli tool or as a set of CURL commands, so that ET can be embedded into containers as well.

**Comment by yarikoptic** [comment]
+1 on this, in particular for CLI.  Wanted to use it within a shell script, checking version according to github releases.

**Comment by satra** [comment]
current worst case scenario, you could do this:

`python -c "import etelemetry as et; et.check_available_version('nipy/nipype', '1.2.1')"`

**Comment by yarikoptic** [comment]
indeed! to say the truth I forgot the obvious answer ;-)  for me it should be enough for now
